src/main/java/pl/sda/javapoz/model/Pagination.java

```diff
@@ -1,8 +1,8 @@
 package pl.sda.javapoz.model;
 
-/**
- * Created by RENT on 2017-03-25.
- */
+import javax.persistence.Entity;
+
+@Entity
```

Comment:
Ta klasa dzieliła newsy na stronie na kilka mmniejszych stron. mozna było chyba wybrac ile newsów ma się wyswietlić, Też to raczej nie jest encja

Reply:
Wyjasnione